src/Modules/UngradedModules/Presentation.jl

```diff
@@ -559,6 +563,50 @@ function prune_with_map(M::ModuleFP{T}) where {T<:MPolyRingElem{<:FieldElem}} #
   return M_new, phi
 end
 
+function _presentation_minimal(SQ::ModuleFP{T}) where {T<:MPolyRingElem{<:FieldElem}}
+  R = base_ring(SQ)
+
+  # Prepare to set some names
+  br_name = AbstractAlgebra.get_name(R)
+  if br_name === nothing
+    br_name = "br"
+  end
+  
+  SQ_new, phi = prune_with_map(SQ)
+  F0 = ambient_free_module(SQ_new)
+
+  # M_new is a quotient of a free module
+  proj_map = hom(F0, SQ_new, gens(SQ_new), check=false)
+  F0_to_SQ = compose(proj_map, phi)
+  F0_to_SQ.generators_map_to_generators = true
+  AbstractAlgebra.set_name!(F0, "$br_name^$(ngens(SQ.sub))")
```

Comment:
Why do you ned to set this name here and below? Since F0 and F1 are free module anyway, couldn't this be done generally as part of the `show` methods for free modules?

As it is, this hardcodes the name of the basering, so if it changes later on, it will look ugly. Likewise if there is no name set, you hardcode `br` forever.

Reply:
This is the current standard for presentations/resolutions/etc. and the same at other places. Thus I would keep it like this here right now and would instead think of some way to replace all of these instances by some AA printing magic in the future.

Reply:
Yeah if this is pervasive we should not try to deal with it in this PR. But perhaps we can open an issue as a reminder? 

Or perhaps we just add it to https://github.com/oscar-system/Oscar.jl/issues/2891